requirements.txt

```diff
@@ -3,3 +3,4 @@ attrs>=21.2.0
 defusedxml>=0.7.1
 httpx>=0.21.0
 netifaces>=0.11.0
+async-timeout>=4.0.2
```

Comment:
```suggestion
```

Reply:
Resolved both the requirements and setup in https://github.com/dcmeglio/denonavr/commit/b5ab97ff9b19dfb8ec661e426694b372cb154449